docs/docs/guides/labs/components/building-pipelines-with-components/creating-a-project-with-components.md

```diff
@@ -9,7 +9,7 @@ import Preview from '@site/docs/partials/\_Preview.md';
 
 After [installing dependencies](/guides/labs/components#installation), you can scaffold a components-ready project. In the example below, we scaffold a project called `jaffle-platform`:
 
-<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-scaffold.txt" />
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-a-uv-scaffold.txt" />
 
 This command builds a project and initializes a new Python virtual environment inside of it. When using `dg`'s default environment management behavior, you won't need to worry about activating this virtual environment yourself.
 
```

Comment:
We should note somewhere in this file that the project itself is installed as an editable install

Reply:
in the tabs in "Scaffold a new project" section